src/PythonClient/src/quixstreams/models/timeseriesbuffer.py

```diff
@@ -75,14 +74,23 @@ def _finalizerfunc(self):
     @property
     def on_data_released(self) -> Callable[[Union['StreamConsumer', 'StreamProducer'], TimeseriesData], None]:
         """
-        Gets the handler for when the stream receives data. First parameter is the stream the data is received for, second is the data in TimeseriesData format.
+        Gets the handler for when the stream receives data. First parameter is the stream the data is received for,
```

Comment:
There is a bit of inconsistence with the handling of segment "the first parameter ... ".
Most other classes have it in the return statement or in the args statement, not in the property description.